### DISCUSSION TITLE
Chromium Manifest v3

### DISCUSSION THREAD

**Discussion by oSumAtrIX** [ramensoftware/windhawk-mods#834] [Mod ideas]
As we know, Manifest v3 will soon be forced on all Chromium browsers.
However, the **effective** issue with Manifest v3 is what [uBlock Origin Lite highlights](https://chromewebstore.google.com/detail/ublock-origin-lite/ddkjiahejlhfcafbddmgiahcphecmpfh?hl=en):

![image](https://github.com/user-attachments/assets/eb2c9be9-4045-4d60-874c-8cf37119ae31)

That said, assuming a Windhawk mod could accept that request automatically (just for uBlock Origin Lite or any preconfigured extension making that request) and these extensions requesting this permission by default would effectively disable the side effects of Manifest v3.

**Comment by m417z** [comment]
Interesting. Just below the text from your screenshot, the following is said:

> You can set the default filtering mode from uBOL's options page. If you pick the Optimal or Complete mode as the default one, you will need to grant uBOL the permission to read and modify data on all websites.

Isn't this what you're after? Or do you prefer having it in basic mode by default, but would like to select other modes for selected websites without seeing the permission popup? Even then, once you choose the Complete mode from the settings and accept the permission, the popup doesn't seem to be displayed even if you change the default mode back to Basic.

**Reply by oSumAtrIX** [reply]
In that case the mod idea is obsolete, thanks.

**Comment by oSumAtrIX** [comment]
This is possible:

![image](https://github.com/user-attachments/assets/1cf8f56c-002a-46c6-acc5-096d341d02ff)
